go.mod

```diff
@@ -293,3 +294,5 @@ require (
 )
 
 replace github.com/cs3org/go-cs3apis => github.com/c0rby/go-cs3apis v0.0.0-20230110100311-5b424f1baa35
+
+replace github.com/leonelquinteros/gotext => github.com/kobergj/gotext v0.0.0-20230309141732-b909eb0b8956
```

Comment:
ok, did you open an upstream PR from that branch?

Reply:
Not yet. Struggling with unit tests. Will come soon